apps/src/blockly/addons/functionEditor.js

```diff
@@ -29,74 +27,3 @@ export default class FunctionEditor {
   // TODO
   openWithNewFunction() {}
 }
-
```

Comment:
we may end up with merge conflicts with the modal function editor here, but I think the same functions were deleted in both prs